device/mellanox/x86_64-mlnx_msn2700-r0/platform.json

```diff
@@ -0,0 +1,999 @@
+                "low_threshold": 10,
+                "low_critical_threshold": 0
+            },
+            {
+                "name": "thermal 35",
+                "presence": "true",
+                "model": "FIXME",
+                "serial": "FIXME",
+                "status": "true",
+                "temperature": 30.125,
+                "high_threshold": 100,
+                "high_critical_threshold": 100,
+                "low_threshold": 10,
+                "low_critical_threshold": 0
+            },
+            {
+                "name": "thermal 36",
+                "presence": "true",
+                "model": "FIXME",
+                "serial": "FIXME",
+                "status": "true",
+                "temperature": 30.125,
+                "high_threshold": 100,
+                "high_critical_threshold": 100,
+                "low_threshold": 10,
+                "low_critical_threshold": 0
+            },
+            {
+                "name": "thermal 37",
+                "presence": "true",
+                "model": "FIXME",
+                "serial": "FIXME",
+                "status": "true",
+                "temperature": 30.125,
+                "high_threshold": 100,
+                "high_critical_threshold": 100,
+                "low_threshold": 10,
+                "low_critical_threshold": 0
+            },
+            {
+                "name": "thermal 38",
+                "presence": "true",
+                "model": "FIXME",
+                "serial": "FIXME",
+                "status": "true",
+                "temperature": 30.125,
+                "high_threshold": 100,
+                "high_critical_threshold": 100,
+                "low_threshold": 10,
+                "low_critical_threshold": 0
+            },
+            {
+                "name": "thermal 39",
+                "presence": "true",
+                "model": "FIXME",
+                "serial": "FIXME",
+                "status": "true",
+                "temperature": 30.125,
+                "high_threshold": 100,
+                "high_critical_threshold": 100,
+                "low_threshold": 10,
+                "low_critical_threshold": 0
+            },
+            {
+                "name": "thermal 40",
+                "presence": "true",
+                "model": "FIXME",
+                "serial": "FIXME",
+                "status": "true",
+                "temperature": 30.125,
+                "high_threshold": 100,
+                "high_critical_threshold": 100,
+                "low_threshold": 10,
+                "low_critical_threshold": 0
+            }
+        ],
+        "sfps": [
+            {
+                "name": "sfp 1",
+                "presence": "true",
+                "model": "FIXME",
+                "serial": "FIXME",
+                "status": "true",
+                "lp_mode": "false",
+                "temperature": 30.125,
+                "voltage": 13.1
```

Comment:
`model` and `serial` are dependent upon the actual transceiver connected to the port. Since this can vary, it is not a truth of the platform itself. `temperature` and `voltage` are also variables, so I'm not sure they belong here.

Reply:
fixed